### DISCUSSION TITLE
Primevue MenuBar > Submenu

### DISCUSSION THREAD

**Discussion by alessandro-de-marco** [primefaces/community#511] [PrimeVue]
Hi all,
I am using a MenuBar in PassThrough mode. I started with a basic example taken from the site, but in all examples, the menu is always aligned left. I have moved the menubar to the right, but I am experiencing problems with the submenu, which seems to have wrong styling and not sure how to change that orientation.
<img width="631" alt="Screenshot 2023-11-21 at 00 01 11" src="https://github.com/primefaces/community/assets/84577621/ed7cdc9c-df34-4a89-b213-eb46c93a7e63">

Any Idea on how to fix that?

**Comment by FlipWarthog** [comment]
I don't think there's currently a way to flip the orientation based on a question from another user. It would probably be a feature request to add something like that.
